Lib/fontTools/varLib/models.py

```diff
@@ -530,6 +536,14 @@ def interpolateFromMastersAndScalars(self, masterValues, scalars, *, round=noRou
         deltas = self.getDeltas(masterValues, round=round)
         return self.interpolateFromDeltasAndScalars(deltas, scalars)
 
+    @staticmethod
+    def interpolateFromMastersAndMasterScalars(masterValues, masterScalars):
+        """Interpolate from master-values and master-scalars fetched
+        from getMasterScalars()."""
+        return VariationModel.interpolateFromValuesAndScalars(
```

Comment:
why can't simply add this `interpolateFromValuesAndScalars` staticmethod? What's the use of `interpolateFromMastersAndMasterScalars` -- besides the questionable name itself?
I think "value" is generic enough to mean "source" or "master" or even "delta", it's just the responsibility of the caller to pass in type of scalars that matches the kind of "values".

Reply:
Sounds good to me. I'll adjust.